packages/wallet/Account.js

```diff
@@ -0,0 +1,604 @@
+ */
+class Account {
+  /**
+  * constructor
+  * @param {string} network
+  */
+  constructor(network) {
+    switch (network) {
+      case 'stellar':
+        this.network = 'stellar';
+        this.testAddress = 'https://horizon-testnet.stellar.org/accounts/';
+        StellarSdk.Network.useTestNetwork();
+        break;
+      case 'ethereum':
+        this.network = 'ethereum';
+        break;
+      default:
+        console.log('The name of the network must be stellar or ethereum.');
+    }
+  }
+
+  /**
+   * This is for stellar only
+   * @param {string} assetName
+   * @param {string} issuerPublickey
+   * @param {string} limit
+   * @returns {object|null} xdr.ChangeTrustOp if succeeds or null otherwise
+   */
+  async changeTrust(assetName, issuerPublickey, limit) {
+    switch (this.network) {
+      case 'stellar': {
+        const newAsset = new StellarSdk.Asset(assetName, issuerPublickey);
+        const receiver = await server.loadAccount(this.getAddress());
+        const transaction = new StellarSdk.TransactionBuilder(receiver)
+          .addOperation(StellarSdk.Operation.changeTrust({
+            asset: newAsset,
+            limit
+          }))
+          .build();
+        transaction.sign(this.account);
+        return server.submitTransaction(transaction);
+      }
+      case 'ethereum':
+        console.log('Change trust is not for ethereum.');
+        break;
+      default:
+        console.log('The network is not set correctly.');
+    }
+    return null;
+  }
+
+  /**
+  * set the balance field of this account
+  * @param {string} publicKey - the publick key of the account
+  */
+  async loadBalance(publicKey) {
+    try {
+      const response = await rp(this.testAddress + publicKey);
+      this.balance = JSON.parse(response).balances.slice(-1)[0].balance;// XLM balance
+    } catch (err) {
+      console.log('Error in loading balance');
+    }
+  }
+
+  /**
+   * Set the account field of this account to be the argument;
+   * For Stellar, upload it to the testnet and update the balance;
+   * For Ethereum, update its balance read on the Rinbeky testnet.
+   * @param {object} account - A stellar or ethereum account
+   */
+  async setAccount(originalAccount) {
+    switch (this.network) {
+      case 'stellar':
+        this.account = originalAccount;
+        await this.loadBalance(this.getAddress());
+        if (this.balance === '0.0000000' || this.balance === undefined) {
+          try {
+            await rp({
+              url: 'https://friendbot.stellar.org',
+              qs: { addr: this.getAddress() },
+              json: true
+            });
+            await this.loadBalance(this.getAddress());
+          } catch (err) {
+            console.log(err);
+          }
+        }
+        break;
+      case 'ethereum': {
+        this.account = originalAccount;
+        const response = await web3.eth.getBalance(originalAccount.address);
```

Comment:
`originalAccount.address` should be `this.account.address` for consistency

Reply:
changed accordingly